src/Phoenix/PhoenixClient.ts

```diff
@@ -86,13 +96,32 @@ export class PhoenixClient {
             status: ContainerStatusType.Active,
           };
         } else if (response.status === HttpStatusCodes.NotFound) {
+          const error = "Disconnected from compute workspace";
+          Logger.logError(error, "PhoenixClient/ContainerStatus");
```

Comment:
>PhoenixClient/ContainerStatus [](http://example.com/codeflow?start=34&length=29)

dont need this - can u pas empty string here?

for customer - PhoenixClient is not something meaningful.